### PR TITLE
POM-292 Introduce PolicyLink component. Use in footer and ad reply.

### PR DESCRIPTION
**Pull request by ermenegildo500**
The component renders html link to a pdf in the correct
language, and with the label in correct gramatical case.

The component is used in:
* footer
* offer reply for a not logged user.

### CODE REVIEW

File: src/app/shared/components/policy-link/policy-link.component.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<a
+  [href]="'https://pomagamukrainie.gov.pl/assets/rodo/politykaprywatnosci' + getSuffixForTranslatedResource() + '.pdf'"
+  target="_blank"
+  class="navbar-item"
+>
+  <ng-template [ngIf]="denominator">{{ "POLICY" | translate }}</ng-template>
+  <ng-template [ngIf]="!denominator">{{ "PRIVATE_POLICY" | translate }}</ng-template>
```
Comment (mgol) [inline src/app/shared/components/policy-link/policy-link.component.html:7]: How about:
```suggestion
  <ng-template>{{ (denominator ? "POLICY" : "PRIVATE_POLICY") | translate }}</ng-template>
```

File: src/app/shared/components/policy-link/policy-link.component.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+@Component({
+  selector: 'app-policy-link',
+  templateUrl: './policy-link.component.html',
+  styleUrls: ['./policy-link.component.scss'],
+})
+export class PolicyLinkComponent {
+  corePath = CorePath;
+
+  // TODO: introduce grammatical cases, starting with denominator and accusative.
+  @Input() denominator: boolean = true;
```
Comment (mgol) [inline src/app/shared/components/policy-link/policy-link.component.ts:17]: Be careful with generalizing it; different languages may have different grammar cases and different rules when to use which. In general, we should instead start using proper translation tools; this is just a hack for MVP1.

**Review by mgol — APPROVED** [review]
Looks good, I just have one suggestion.

**Review by mgol — APPROVED** [review]

File: src/app/shared/components/policy-link/policy-link.component.html (hunk 1 above)
Comment (ermenegildo500) [reply src/app/shared/components/policy-link/policy-link.component.html]: Right, thanks! updated in d4fd02f2, even shorter.

File: src/app/shared/components/policy-link/policy-link.component.ts (hunk 2 above)
Comment (ermenegildo500) [reply src/app/shared/components/policy-link/policy-link.component.ts:17]: Ack. +1. I propose not to address in this PR.

**Pull request merged**